src/dos/program_mount.cpp

```diff
@@ -57,21 +57,21 @@ void MOUNT::Move_Z(char new_z)
 		Drives[25]      = nullptr;
 		if (!first_shell) return; //Should not be possible
 		/* Update environment */
-		std::string line = "";
+		std::string value   = "";
 		std::string tempenv = {new_drive_z, ':', '\\'};
 		if (auto result{first_shell->GetEnvStr("PATH")}) {
-			line = std::move(*result);
-			std::string::size_type idx = line.find('=');
-			std::string value = line.substr(idx +1 , std::string::npos);
-			while ( (idx = value.find("Z:\\")) != std::string::npos ||
-				(idx = value.find("z:\\")) != std::string::npos  )
-				value.replace(idx,3,tempenv);
-			line = std::move(value);
+			value = std::move(*result);
```

Comment:
Looks like we were moving already... okay, hopefully this won't result in hard to debug bugs down the road 🤞🏻